### ISSUE TITLE
bytes.Compare doesn't work for wasm

### ISSUE THREAD

**Issue by asdine** [tinygo-org/tinygo#1034]
Using `bytes.Compare` causes a runtime error:

```go
package main

import "bytes"

func main() {
	bytes.Compare([]byte{1}, []byte{1})
}
```

Error:

```
Uncaught (in promise) LinkError: WebAssembly.instantiate(): Import #0 module="env" function="internal/bytealg.Compare" error: function import requires a callable
```

Compiled using the 0.12.0 docker  image:

```
docker run ... \
    tinygo/tinygo:0.12.0 tinygo build -o /dist/exp.wasm -target wasm --no-debug exp
```

**Comment by jaddr2line** [comment]
This will be fixed by #1036.

**Comment by niaow** [comment]
This has been fixed by #1036.
